platforms/web/src/App.tsx

```diff
@@ -51,11 +52,34 @@ function Button({ onClick, imagePath, alt, state }: ButtonProps) {
 }
 
 function App() {
-    const { ref, isWysiwygReady, wysiwyg, formattingStates, debug } =
+    const [enterToSend, setEnterToSend] = useState(true);
+
+    const inputEventProcessor = (
+        e: WysiwygInputEvent,
+        wysiwyg: Wysiwyg,
+    ): WysiwygInputEvent | null => {
+        if (e instanceof ClipboardEvent) {
+            return e;
+        } else if (enterToSend && e.inputType === 'insertParagraph') {
+            console.log(`SENDING: ${wysiwyg.content()}`);
+            e.preventDefault();
+            e.stopPropagation();
+            wysiwyg.actions.clear();
```

Comment:
I wonder if your proposal with changing the `inputType` to customise the behaviour is better here

Reply:
You mean add a new `inputType` that tells us to clear? I don't think there is an existing one that means that.

Reply:
We have a custom one, you can take a look in `processInput`

Reply:
But you think we should create a new custom event to clear the composer, instead of calling `.clear()` here? Why is that better?

Reply:
To be honest, I don't know.

Better to keep your implementation, it's clearer and it works